oregon-b/snippets-stage/deploy.yaml

```diff
@@ -38,278 +37,157 @@ spec:
+              name: snippets-stage-secrets
         - name: AWS_STORAGE_BUCKET_NAME
           valueFrom:
             secretKeyRef:
               key: aws-storage-bucket-name
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: BUNDLE_BROTLI_COMPRESS
-          valueFrom:
-            secretKeyRef:
-              key: bundle-brotli-compress
-              name: snippets-stage-v191-env
+          value: "True"
         - name: CACHALOT_ENABLED
-          valueFrom:
-            secretKeyRef:
-              key: cachalot-enabled
-              name: snippets-stage-v191-env
+          value: "False"
         - name: CACHE_URL
           valueFrom:
             secretKeyRef:
               key: cache-url
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: CSP_REPORT_ENABLE
-          valueFrom:
-            secretKeyRef:
-              key: csp-report-enable
-              name: snippets-stage-v191-env
+          value: "True"
         - name: CSP_REPORT_ONLY
-          valueFrom:
-            secretKeyRef:
-              key: csp-report-only
-              name: snippets-stage-v191-env
+          value: "True"
         - name: CSV_EXPORT_ROOT
           valueFrom:
             secretKeyRef:
               key: csv-export-root
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: DATABASE_URL
           valueFrom:
             secretKeyRef:
               key: database-url
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: DEAD_MANS_SNITCH_DISABLE_SNIPPETS
           valueFrom:
             secretKeyRef:
               key: dead-mans-snitch-disable-snippets
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: DEAD_MANS_SNITCH_PRODUCT_DETAILS
           valueFrom:
             secretKeyRef:
               key: dead-mans-snitch-product-details
-              name: snippets-stage-v191-env
+              name: snippets-stage-secrets
         - name: DEBUG
-          valueFrom:
-            secretKeyRef:
-              key: debug
-              name: snippets-stage-v191-env
-        - name: DEIS_APP
-          valueFrom:
-            secretKeyRef:
-              key: deis-app
-              name: snippets-stage-v191-env
-        - name: DEIS_DOMAIN
-          valueFrom:
-            secretKeyRef:
-              key: deis-domain
-              name: snippets-stage-v191-env
+          value: "False"
         - name: ENABLE_ADMIN
-          valueFrom:
-            secretKeyRef:
-              key: enable-admin
-              name: snippets-stage-v191-env
+          value: "True"
         - name: ENFORCE_HOST
-          valueFrom:
-            secretKeyRef:
-              key: enforce-host
-              name: snippets-stage-v191-env
+          value: "snippets.allizom.org"
```

Comment:
As we discussed earlier on zoom, this is the current setting in stage, but neither dev nor prod have this setting and I'm not sure if this has any effect given that `ALLOWED_HOSTS=*`. This appears to be a leftover from testing and is likely something we should remove-- @glogiotatidis can you confirm?